index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+          <ul class="navbar-flex">
+            <li class="nav-logo"><a href="#">Maryam</a></li>
+            <li class="nav-logo">
+              <a href="#"><i class="fa-solid fa-bars"></i></a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      
+      <section class="main-div">
+        <h1 class="main-heading">Hello There. I'm Maryam</h1>
+        <h3 class="support-text">I'am a Software Developer</h3>
+        <p class="descriptive-text">
+          I can help you build a product , feature or website Look through some of
+          my work and experience! If you like what you see and have a project you
+          need coded, don't hesitate to contact me.
+        </p>
+        <ul class="accounts">
+          <li>
+            <a href="#"><img src="/images/git.png" alt=""></a>
+          </li>
+          <li>
+            <a href="#"><img src="/images/in.png" alt=""></a>
+          </li>
+          <li>
+            <a href="#"><img src="/images/al.png" alt=""></a>
+          </li>
+          <li>
+            <a href="#"><img src="/images/twi.png" alt=""></a>
+          </li>
+          <li>
+            <a href="#"><img src="/images/med.png" alt=""></a>
```

Comment:
- [OPTIONAL] I would highly recommend that give those `img`  a more descriptive message because it will make your HTML semantic